src/patchelf.cc

```diff
@@ -2505,6 +2508,7 @@ static void showHelp(const std::string & progName)
   [--clear-execstack]\n\
   [--set-execstack]\n\
   [--rename-dynamic-symbols NAME_MAP_FILE]\tRenames dynamic symbols. The map file should contain two symbols (old_name new_name) per line\n\
+  [--no-clobber-old-sections]\t\tDo not clobber old section values - only use when executable does self-referential tricks.\n\
```

Comment:
What are self-referential tricks? Can you add more details here?

Reply:
Basically anything that relies on things remaining at the same offset in the binary.